python/tool_base/Impacts.py

```diff
@@ -204,6 +204,25 @@ def run_method(self):
 
         print("Have parameters: " + str(len(paramList)))
 
+        varList = utils.list_from_workspace(ws, "w", "variables")
+        if self.args.setParameters is not None:
+            set_parameters = self.args.setParameters.split(",")
+            set_parameters_str = ""
+            for ind, setParam in enumerate(set_parameters):
+                if "rgx{" in setParam:
+                    eqs_to = setParam.split("=")[-1]
+                    pattern = setParam.split("=")[0]
+                    pattern = pattern.replace("'rgx{", "").replace("}'", "")
+                    pattern = pattern.replace("rgx{", "").replace("}", "")
+                    set_parameters[ind] = ""
+                    for var in varList:
+                        if re.search(pattern, var):
+                            var_str = var + "=" + eqs_to
+                            set_parameters_str += var_str + ","
+                else:
+                    set_parameters_str += setParam + ","
+        self.args.setPhysicsModelParameters = set_parameters_str.rstrip(",")
```

Comment:
I think this is meant to be inside the if block starting on l.208, or else `set_parameter_str` doesn't exist.

Reply:
Thanks for spotting this!